.circleci/config.yml

```diff
@@ -872,7 +927,19 @@ workflows:
             - lint-checks
       - end-to-end-mobile-test-android
       - end-to-end-mobile-test-ios
-      - protocol-test:
+      - protocol-test-common:
+          requires:
```

Comment:
Just a question, but this will run the same lint in every parallelised batch run, or it's just running once?
(even if it runs multiple times it better than what we have, hahaha, so great!)

Reply:
This just means that the job won't run until the 'lint-checks' job passes. So we're only running the linter once